src/pages/SetPasswordPage.js

```diff
@@ -1,21 +1,22 @@
 import React, {Component} from 'react';
 import {
-    SafeAreaView,
     Text,
     TextInput,
     View,
+    SafeAreaView,
```

Comment:
NAB, this change isn't necessary. Always nice to avoid changes that just create extra noise for ppl when reviewing diffs.